apps/inkbeard/src/components/__tests__/AppIcon.spec.js

```diff
@@ -3,7 +3,7 @@ import { mount } from '@vue/test-utils';
 import AppIcon from '../AppIcon.vue';
 
 describe('AppIcon', () => {
-  it('renders classes with default large style', () => {
+  it('should render classes with default large style', () => {
```

Comment:
This was showing a colleague  how I would normally phrase unit test to use testing as a form of documentation.